airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,271 @@
+
+    list_operations >> get_operation
+
+    # [START howto_operator_gcp_transfer_resume_operation]
+    resume_operation = GcpTransferServiceOperationResumeOperator(
+        task_id="resume_operation", operation_name="{{task_instance.xcom_pull('get_operation')['name']}}"
+    )
+    # [END howto_operator_gcp_transfer_resume_operation]
+
+    get_operation >> resume_operation
+
+    # [START howto_operator_gcp_transfer_wait_operation]
+    wait_for_operation_to_end = GCPTransferServiceWaitForJobStatusSensor(
+        task_id="wait_for_operation_to_end",
+        job_name="{{task_instance.xcom_pull('create_transfer_job_from_aws')['name']}}",
+        project_id=GCP_PROJECT_ID,
+        expected_statuses={GcpTransferOperationStatus.SUCCESS},
+        poke_interval=WAIT_FOR_OPERATION_POKE_INTERVAL,
+    )
+    # [END howto_operator_gcp_transfer_wait_operation]
+
+    resume_operation >> wait_for_operation_to_end
+
+    job_time = datetime.utcnow() + timedelta(minutes=2)
+
+    gcs_to_gcs_transfer_body['schedule']['startTimeOfDay'] = (datetime.utcnow() + timedelta(minutes=2)).time()
+
+    create_transfer_job_from_gcp = GcpTransferServiceJobCreateOperator(
+        task_id="create_transfer_job_from_gcp", body=gcs_to_gcs_transfer_body
+    )
+
+    wait_for_operation_to_end >> create_transfer_job_from_gcp
+
+    wait_for_second_operation_to_start = GCPTransferServiceWaitForJobStatusSensor(
+        task_id="wait_for_second_operation_to_start",
+        job_name="{{ task_instance.xcom_pull('create_transfer_job_from_gcp')['name'] }}",
+        project_id=GCP_PROJECT_ID,
+        expected_statuses={GcpTransferOperationStatus.IN_PROGRESS},
+        poke_interval=WAIT_FOR_OPERATION_POKE_INTERVAL,
+    )
+    create_transfer_job_from_gcp >> wait_for_second_operation_to_start
+
+    # [START howto_operator_gcp_transfer_cancel_operation]
+    cancel_operation = GcpTransferServiceOperationCancelOperator(
+        task_id="cancel_operation",
+        operation_name="{{task_instance.xcom_pull("
+        "'wait_for_second_operation_to_start', key='sensed_operations')[0]['name']}}",
+    )
+    # [END howto_operator_gcp_transfer_cancel_operation]
+
+    wait_for_second_operation_to_start >> cancel_operation
+
+    # [START howto_operator_gcp_transfer_delete_job]
+    delete_transfer_from_aws_job = GcpTransferServiceJobDeleteOperator(
+        task_id="delete_transfer_from_aws_job",
+        job_name="{{task_instance.xcom_pull('create_transfer_job_from_aws')['name']}}",
+        project_id=GCP_PROJECT_ID,
+    )
+    # [END howto_operator_gcp_transfer_delete_job]
+
+    cancel_operation >> delete_transfer_from_aws_job
+
+    delete_transfer_from_gcp_job = GcpTransferServiceJobDeleteOperator(
+        task_id="delete_transfer_from_gcp_job",
+        job_name="{{task_instance.xcom_pull('create_transfer_job_from_gcp')['name']}}",
+        project_id=GCP_PROJECT_ID,
+    )
+
+    delete_transfer_from_aws_job >> delete_transfer_from_gcp_job
```

Comment:
Can we add all task dependencies at the end of file, please? It is much easier to read and makes it uniform with other examples

Reply:
I updated a PR.